### PR TITLE
quacs: add git hashes to footer for site, data. Resolves #390,#389

### LINKED ISSUE #390: Add hash of commit which was used to build QuACS to the footer

### PR DESCRIPTION
**Pull request by johnnyapol**

### CODE REVIEW

**Comment by johnnyapol** [comment]
image
Here is a demo - there are clickable hyperlinks for both the site and data

File: site/scripts/build_single.sh [hunk 1]
```diff
@@ -42,6 +42,12 @@ echo Setting quacs-rs to build for "$CURR_SEMESTER"
 mkdir src/quacs-rs/src/data || rm -rf src/quacs-rs/src/data/*
 cp src/store/data/semester_data/$CURR_SEMESTER/*.rs src/quacs-rs/src/data
 
+QUACS_COMMIT_HASH=$(git rev-parse HEAD)
+QUACS_DATA_COMMIT_HASH=$(cd src/store/data/semester_data && git rev-parse HEAD)
```
Comment (benjaminrsherman) [inline site/scripts/build_single.sh:46]: I think this would be cleaner using `git -C` instead of `cd`

File: site/src/App.vue [hunk 2]
```diff
@@ -146,7 +146,15 @@
...
-      <div class="footer-updated">Last updated {{ lastUpdated }}</div>
+      <div class="footer-updated">
+        Last updated {{ lastUpdated }} (<a
+          :href="'https://github.com/quacs/quacs/commit/' + quacsHash"
+          >site</a
+        >,
+        <a :href="'https://github.com/quacs/quacs-data/commit/' + dataHash"
+          >data</a
+        >)
+      </div>
```
Comment (elihschiff) [inline site/src/App.vue:157]: Can you give these links `color: var(--raw-link)`?
I dont care if you use style tags or a class, but they are just hard to read on dark mode with the default color

**Review by elihschiff — CHANGES_REQUESTED** [review]

**Review by benjaminrsherman — CHANGES_REQUESTED** [review]

**Comment by johnnyapol** [comment]
Updated 
image

**Review by benjaminrsherman — APPROVED** [review]

**Review by elihschiff — APPROVED** [review]

**Pull request merged**